### PR TITLE
New format in context popup for lat/lon coordinate

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

File: src/components/contextpopup/partials/contextpopup.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
               <td>{{coord2056}}</td>
           </tr>
           <tr>
-              <td>WGS 84 (long/lat)</td>
+              <td>WGS 84 (lat/lon)</td>
               <td>{{coord4326}}</td>
```
Comment (oterral) [inline src/components/contextpopup/partials/contextpopup.html]: I think @davidoesch wants 2 lines : one with normal coordinates and one with the `toStringHDMS` transformation.

**Comment by davidoesch** [comment]
Yep : link to test?

**Comment by pfanguin** [comment]
Test link : https://mf-geoadmin3.dev.bgdi.ch/coordinate_format/

**Comment by davidoesch** [comment]
Remove the space in lat and in lon . only between lat and lon a space 50°40'46,461"N 95°48'26,533"W

**Comment by pfanguin** [comment]
It's done.

**Comment by davidoesch** [comment]
Hmm to be accurate enough: add three digits egg
47°13′08.123″N 8°48′28.567″E to have a good precision on tlm level

**Comment by gjn** [comment]
@pfanguin adaptions still wanted.

**Comment by pfanguin** [comment]
it's done, you can test with the same link

**Comment by davidoesch** [comment]
no its not ok
in tooltip and footer it is currently 47°09′09″N 7°54′48″E
expected 47°09′09.123″N 7°54′48.456″E
-> three digits precision

**Comment by pfanguin** [comment]
Yes, Because the deploy of my branch failed.

**Comment by davidoesch** [comment]
whatever you dont push to prod ;-)

**Comment by oterral** [comment]
@pfanguin  prepare a PR for OpenLayers.

**Comment by gjn** [comment]
@davidoesch Can we live with the 2-precision solution until openlayers accepts our (yet to be done) PR?

**Comment by pfanguin** [comment]
PR ol3 see here : https://github.com/openlayers/ol3/pull/4787

**Comment by davidoesch** [comment]
Yes @gjin 2 dig OK for now

**Comment by oterral** [comment]
Currently we have 0 digits not 2. The @pfanguin allows to have more than 0 digits.

**Comment by gjn** [comment]
So there's no way to get 2/3 digits without ol3 fix?

**Comment by oterral** [comment]
I speak digits  after comma.

Currently we have 0 digits: 47°09′09″N 7°54′48″E
With the fix we can have 1 or more digits after comma : 47°09′09.123″N 7°54′48.456″E

**Comment by oterral** [comment]
Still waiting 2 PRs https://github.com/openlayers/ol3-cesium/pull/317 and https://github.com/openlayers/ol3/pull/4845

**Comment by oterral** [comment]
Librairies updated. Test link update too. https://mf-geoadmin3.dev.bgdi.ch/coordinate_format/

**Comment by davidoesch** [comment]
Test link works like a charm on mobile

**Comment by oterral** [comment]
jenkins test

**Comment by oterral** [comment]
Doesn't work on IE9

**Comment by oterral** [comment]
It seems we need to use [polyfill](https://cdn.polyfill.io/v2/docs/) now for ie9, see https://github.com/openlayers/ol3/pull/4866 and https://github.com/openlayers/ol3/issues/4865

@gjn are you ok if we merge this PR ? and I come before the end of week with a PR for polyfill integration.

**Comment by gjn** [comment]
hmm. not sure. Would like to have everything in one PR.

Also, be carefull with polyfill, we have some custom adaptions already for IE9 (see IE9 specific files) and other IE browsers.

**Comment by oterral** [comment]
Ok i do it in this PR.

**Comment by oterral** [comment]
Maybe with polyfill we can remove some specific stuff too.

**Comment by gjn** [comment]
Probably, but looking at https://github.com/geoadmin/mf-geoadmin3/blob/master/src/lib/IE9Fixes.js makes me doubt that :)

**Comment by oterral** [comment]
Sure, I will add the minimal polyfill we need

**Comment by oterral** [comment]
Tests ok!!!!!

**Comment by oterral** [comment]
@gjn can we merge this one ?

**Pull request merged**